### ISSUE TITLE
Forgetting parameter for attack 0 or 1 give an erroneous error message

### ISSUE THREAD

**Issue by aircrack-ng** [aircrack-ng/aircrack-ng#125]
**Reported by jeroenimo on 2 Mar 2007 08:08 UTC**

When aireplay-ng is started with the -0 option (--deauth) without the amount of times the attack should be done there is an error wich not tells the user that he's forgetting something and says :

```
ioctl(SIOCGIFINDEX) failed: 
                 No such device error
```

**Comment by aircrack-ng** [comment]
**Comment by anonymous on 2 Mar 2007 08:08 UTC**

Replying to [jeroenimo](ticket:125):
> When aireplay-ng is started with the -0 and the -1 option (--deauth) without the amount of times the attack should be done there is an error wich not tells the user that he's forgetting something and says : ioctl(SIOCGIFINDEX) failed: 
>                  No such device error

**Comment by aircrack-ng** [comment]
**Modified by misterx on 2 Mar 2007 08:08 UTC**

**Comment by aircrack-ng** [comment]
**Comment by misterx on 2 Mar 2007 08:08 UTC**

(In #209) Give a better error message when the user forget to give the parameter to attack 0 or 1 (Closes: #125).

**Comment by aircrack-ng** [comment]
**Modified by misterx on 2 Mar 2007 08:08 UTC**
